kvmagent/kvmagent/test/http/conftest.py

```
@@ -0,0 +1,743 @@
+        subprocess.run(
+            ['sshpass', '-p', password, 'scp', '-o', 'StrictHostKeyChecking=no',
+             tmp, 'root@%s:%s' % (host, remote_path)],
+            capture_output=True, timeout=30, check=True,
+        )
+    finally:
+        os.unlink(tmp)
+
+
+def _scp_from_remote(host, password, remote_path, local_path):
+    """SCP a file from remote to local."""
+    subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)